Gemfile

```diff
@@ -2,8 +2,9 @@ source 'https://rubygems.org'
 
 gem 'rails', '~> 4.1.6'
 gem 'envied', '~> 0.8.1'
-gem 'pg', '~> 0.17.1'
+gem 'pg', '~> 0.18.0'
 gem 'virtus', '~> 1.0.5'
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw]
```

Comment:
Okay, but it's not project-dependency and shouldn't be included in project's Gemfile, rather it should be configured in your development environment.